### PR TITLE
add luna2 on migaloo

### PR DESCRIPTION
**Pull request by fragwuerdig**

### CODE REVIEW

File: migaloo/assetlist.json [hunk 1]
```diff
@@ -382,6 +382,52 @@
...
+      "name": "Luna",
+      "display": "luna",
+      "symbol": "LUNA",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "terra2",
+            "base_denom": "uluna",
+            "channel_id": "channel-86"
```
Comment (JeremyParish69) [inline migaloo/assetlist.json:408]: Pending registration of this IBC connection. The transfer<>transfer channel should be first in the array of channels, then the wasm connetion after.

Comment (fragwuerdig) [reply migaloo/assetlist.json:408]: There is no Transfer-Transfer channel on this connection...

Comment (fragwuerdig) [reply migaloo/assetlist.json:408]: wait - this literally _is_ a transfer-transfer channel. i don't understand what the problem is right now... can you please elaborate? please note: this is migaloo-terra2 and _NOT_ migaloo-terra

Comment (JeremyParish69) [reply migaloo/assetlist.json:408]: My bad, this is already registered. I had terra in mind

**Pull request merged**